src/failures/src/a320.ts

```diff
@@ -15,6 +15,8 @@ export const A320Failure = Object.freeze({
     YellowReservoirReturnLeak: 29008,
     LeftPfdDisplay: 31000,
     RightPfdDisplay: 31001,
+    FlightWarningComputer1: 31500,
+    FlightWarningComputer2: 31501,
```

Comment:
I chose 50 specifically because the FWS's ATA is 31-50. While see your point I think it's fine to just go with this for now and shuffle these around in future if necessary.

Reply:
Yeah. I figured indeed. 😀